### ISSUE TITLE
Improvements to Feature Request- and Question-Templates.

### ISSUE THREAD

**Issue by Glitchy-Tozier** [florisboard/florisboard#815]
For some reason I can't create new pull-requests so I'm just writing this issue. It's pretty much the same changes we introduced to the other templates.
![image](https://user-images.githubusercontent.com/59611881/117149247-56b11b80-adb7-11eb-8e3a-9cf324abba24.png)

## Feature...
```diff
<!--
- - Describe your idea in a short but concise way.
- - If you have multiple ideas which are not directly connected to other, file an issue per idea. This makes it easy to implement feature proposal at a time.
- - If you have any examples, e.g. screenshots or other keyboards have the proposed feature implemented, link them here.
- - Please search existing proposals to avoid creating duplicates.
- - Thank you for your help in making FlorisBoard better!
-->

<!--
+ Thank you for your help in making FlorisBoard better!
+
+ Guide to a good feature-request:
+ • Please search existing proposals to avoid creating duplicates.
+ • If you have multiple ideas which are not directly connected to other, file a new issue for each idea. This makes it easier to implement your proposals.
+ • Describe your idea in a short but concise way.
+ • If you have any examples, e.g. screenshots or other keyboards have the proposed feature implemented, feel free to post them after your description.
-->
```

Non-diff format: 
```
<!--
Thank you for your help in making FlorisBoard better!

Guide to a good feature-request:
• Please search existing proposals to avoid creating duplicates.
• If you have multiple ideas which are not directly connected to other, file a new issue for each idea. This makes it easier to implement your proposals.
• Describe your idea in a short but concise way.
• If you have any examples, e.g. screenshots or other keyboards have the proposed feature implemented, feel free to post them after your description.
-->
```

## Question:
(I think we can remove the TY-message as this isn't the issue-type for suggesting improvements. If you disagree, I suggest moving the ty-message up to line 1.)
```diff
<!--
- - If you need assistance in using FlorisBoard, ask it here!
- - If you want to suggest an idea for this project, please use Feature request template instead.
- - Please search existing questions to avoid creating duplicates.
- - Thank you for your help in making FlorisBoard better!
-->

<!--
+ If you need assistance in using FlorisBoard, ask it here!
+ 
+ • Please search existing questions to avoid creating duplicates.
+ • If you want to suggest an idea for this project, please use Feature request template instead.
-->
```

Non-diff format:
```
<!--
If you need assistance in using FlorisBoard, ask it here!

• Please search existing questions to avoid creating duplicates.
• If you want to suggest an idea for this project, please use Feature request template instead.
-->
```

**Comment by Glitchy-Tozier** [comment]
ALSO: I noticed that we wrote the "Guide to a good …"-lines like this:
```
Guide to a good bug-report:
Guide to a good crash-report:
Guide to a good feature request:
```
…while the templates themselves are called
```
Bug report
Crash report
Feature request / Suggestion
```
So not sure whether we maybe should remove the dashes. I feel like they look good the way they are, but it's *slightly* inconsistent. Maybe it makes sense though, seeing as how one of them is a heading while the other is regular text.

**Comment by patrickgold** [comment]
Thanks for suggesting the changes to the other templates as well, really appreciate it! Will incorporate your suggested changes once the user dictionaries stuff is ready.

> So not sure whether we maybe should remove the dashes.

I feel like the dashes should be removed to have a bit more consistency in the naming, although I wouldn't even have noticed this without you bringing it up :)

> For some reason I can't create new pull-requests so I'm just writing this issue.

That's strange. If you go to Pull requests and then New, are you able to progress that far? If not then something with the permission level for your user is wrong (although generally every user account has the permission to create pull requests, so not exactly sure here).

**Comment by Glitchy-Tozier** [comment]
> That's strange. If you go to Pull requests and then New, are you able to progress that far?

Yeah, I'm not sure what to do then though. I tried going the braindead route of searching for the file and then clicking the edit-button, but that button was greyed out.
(I'm still a little confused with how exactly GitHub pull-requests work, do you know of any good tutorials?)

---
Off-topic: I think we should release 0.3.11 as soon as possible after the mayor bugs are fixed and all the existing features re-implemented. 0.3.10 is kind of unusable with the constant lags and crashes. I just wanted to mention this as #817 looks like you're already working on something new.

**Comment by patrickgold** [comment]
> Yeah, I'm not sure what to do then though. I tried going the braindead route of searching for the file and then clicking the edit-button, but that button was greyed out.

For users without direct write access the edit button by default forks a project and then allows you to modify the file and make a pr with the changes. If you already have a fork, it could theoretically be that the edit button thus doesn't work, I am not 100% sure though. You could try to delete your FlorisBoard fork and then try out the edit button to verify this theory.

>  (I'm still a little confused with how exactly GitHub pull-requests work, do you know of any good tutorials?)

Not specifically, as I use PRs almost exclusively via the terminal and not via the web UI. A useful guide for me at least is https://gist.github.com/Chaser324/ce0505fbed06b947d962

> Off-topic: I think we should release 0.3.11 as soon as possible after the mayor bugs are fixed and all the existing features re-implemented. 0.3.10 is kind of unusable with the constant lags and crashes.

My release schedule for v0.3.11 is the following:
- May 7th (Friday): v0.3.11-beta05, will contain the changes of #817 
- May 8th (Saturday): v0.3.11-beta06, will fix glide typing and some other bugs not yet cleaned up from the layout rework.
- May 9th (Sunday): v0.3.11 stable release, given no critical bugs are found in beta06. Via GPlay the update should arrive on Tuesday at max, if we are lucky even on Monday evening.

**Comment by Glitchy-Tozier** [comment]
> Not specifically, as I use PRs almost exclusively via the terminal and not via the web UI. A useful guide for me at least is https://gist.github.com/Chaser324/ce0505fbed06b947d962, you could try this one out.

Thank you, I'll check it out as soon as I start working with the terminal!  :)

> My release schedule for v0.3.11 is the following:

Sounds like a plan! Let's hope there aren't any mayor hiccups then!
